superset/assets/webpack.config.js

```diff
@@ -48,7 +49,7 @@ const config = {
       },
       {
         test: /\.jsx?$/,
-        exclude: APP_DIR + '/node_modules',
+        exclude: /(node_modules|bower_components)/,
```

Comment:
I think I hit some problems I may have ended fixing in other ways, going to try to roll this back